code/iaas/api-logic/src/main/java/io/cattle/platform/iaas/api/auth/dao/impl/AuthDaoImpl.java

```diff
@@ -104,7 +103,7 @@ public Account getAccountByUuid(String uuid) {
                 .selectFrom(ACCOUNT)
                 .where(
                         ACCOUNT.UUID.eq(uuid)
-                        .and(ACCOUNT.STATE.eq("active"))
+                        .and(ACCOUNT.STATE.ne("purged"))
```

Comment:
Used by `io.cattle.platform.iaas.api.auth.impl.TokenAccountLookup.getAccount(ApiRequest)`, is this safe?